libdnf5-cli/progressbar/download_progress_bar.cpp

```diff
@@ -71,6 +71,12 @@ std::ostream & operator<<(std::ostream & stream, DownloadProgressBar & bar) {
 void DownloadProgressBar::to_stream(std::ostream & stream) {
     update();
 
+    if (get_state() == ProgressBarState::STARTED && !tty::is_interactive()) {
```

Comment:
What are about other states? I guess ERROR and SUCCESS are final states. WARNING is probably nowhere used. Shouldn't READY be handled here? Wouldn't be better to compare against is_finished()?

Reply:
I've changed the condition to `!is_finished()` anyway. It definitely makes sense.